src/reducers/people.js

```diff
@@ -62,6 +63,14 @@ export default function (state = {}, action) {
         ...state,
         current: {...state.current, memberships}
       }
+    case UPDATE_COMMUNITY_SETTINGS_PENDING:
```

Comment:
I couldn't find a neater way of doing this than catching the action in both the communities and people reducers. But I'm a little suspicious of the duplication...